### ISSUE TITLE
UCUM and UNECE codes are annotations

### ISSUE THREAD

**Issue by dr-shorthair** [qudt/qudt-public-repo#192]
The UCUM and UNECE codes are annotations on QUDT units, so the ontology should reflect this.

**Comment by dr-shorthair** [comment]
Though maybe we should have a discussion about this. The UCUM codes are formally defined and constructed, so they do have 'semantics' even though (in the QUDT context) they are only used as opaque identifiers (i.e. annotations).

**Comment by jhodgesatmb** [comment]
Simon,

Does this mean that we could generate UCUM codes for units that don’t have them? What about the UN/ECE codes? Are they algorithmic as well? I’d just as soon use the real semantics or a link to the real URI if there is one.

Jack

**Comment by dr-shorthair** [comment]
Yes to UCUM - they are algorithmic - see http://unitsofmeasure.org/ucum.html#section-Syntax-Rules 
No for UNECE - it is defined in a static list and the codes are essentially arbitrary - i.e. they really have no semantics, so `qudt:uneceCommonCode` should definitely be a `owl:AnnotationProperty`

**Comment by steveraysteveray** [comment]
Just want to mention that QUDT no longer has any qudt:code property values.

**Comment by dr-shorthair** [comment]
It is still in the 'schema'. Should it be marked `owl:deprecated true` ?

**Comment by steveraysteveray** [comment]
OK, sure. Just added it to my current PR #193

**Comment by steveraysteveray** [comment]
Meanwhile, @dr-shorthair, were you the original contributor of the UCUM codes in QUDT? If so, why are some of them wrapped in []. Should I remove the []'s?

**Comment by steveraysteveray** [comment]
Never mind. I just read the documentation on UCUM that you provided, and see that [] is used for "customary units" and a variety of other purposes. Consider me educated now!
